schema/2.0.0/schema.md

```diff
@@ -151,18 +151,18 @@ The following gene annotation dependencies are *pinned* for this version of the
 
 | Source | Required version | Download |
 |:--|:--|:--|
-| [ENSEMBL (Human)] | Human reference GRCh38 (GENCODE v32/Ensembl 98)] matching [cellranger 2020-A (July 7, 2020) release] | [gencode.v32.primary_assembly.annotation.gtf] |
-| [ENSEMBL (Mouse)] | Mouse reference mm10 (GENCODE vM23/Ensembl 98) matching [cellranger 2020-A (July 7, 2020) release]| [gencode.vM23.primary_assembly.annotation.gtf] |
+| [ENSEMBL (Human)] | Human reference GRCh38 (GENCODE v38/Ensembl 104)] | [gencode.v38.primary_assembly.annotation.gtf] |
+| [ENSEMBL (Mouse)] | Mouse reference GRCm39 (GENCODE vM27/Ensembl 104) | [gencode.vM27.primary_assembly.annotation.gtf] |
 | [ENSEMBL (COVID-19)] | SARS-CoV-2 reference (ENSEMBL assembly: ASM985889v3) | [Sars\_cov\_2.ASM985889v3.101.gtf] |
 | [ThermoFisher ERCC Spike-Ins] | ThermoFisher ERCC RNA Spike-In Control Mixes (Cat # 4456740, 4456739) | [cms_095047.txt] |
 
 [RNA Spike-In Control Mixes]: https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fcms_086340.pdf&title=VXNlciBHdWlkZTogRVJDQyBSTkEgU3Bpa2UtSW4gQ29udHJvbCBNaXhlcyAoRW5nbGlzaCAp
 
 [ENSEMBL (Human)]: http://www.ensembl.org/Homo_sapiens/Info/Index
-[gencode.v32.primary_assembly.annotation.gtf]: http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_32/gencode.v32.primary_assembly.annotation.gtf.gz
+[gencode.v38.primary_assembly.annotation.gtf]: http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_38/gencode.v38.primary_assembly.annotation.gtf.gz
```

Comment:
The **Source** column references ENSEMBL - http://uswest.ensembl.org/Homo_sapiens/Info/Index - but the ftp location is EBI instead of something like http://ftp.ensembl.org/pub/release-104/gtf/homo_sapiens/ ? Can you advise?

Reply:
There's a few minor changes that gencode does to the ENSEMBL gtfs  (see below) . But apart from that, they're essentially identical. Most contributors use the gencode version and per Jason's request we are using the most updated one from gencode.

We have two options:

1. Leave as is since it's technically correct (but confusing).
2. Change the source field to the landing page of gencode: e.g. [human](https://www.gencodegenes.org/human/) 

> The gene annotation is the same in both files. The only exception is that the genes which are common to the human chromosome X and Y PAR regions can be found twice in the GENCODE GTF, while they are shown only for chromosome X in the Ensembl file.
>
> In addition, the GENCODE GTF contains a number of attributes not present in the Ensembl GTF, including annotation remarks, APPRIS tags and other tags highlighting transcripts experimentally validated by the GENCODE project or 3-way-consensus pseudogenes (predicted by Havana, Yale and UCSC). See our complete list of tags for more information.

Reply:
I'd prefer `2`.